packages/language-service/README.md

```diff
@@ -0,0 +1,83 @@
+# `@mdx-js/language-service`
+
+## What is this?
+
+This package implements the logic needed to provide [MDX][] intellisense.
+This is done by wrapping the [TypeScript][] language service.
+
+## When should I use this?
+
+This package is intended for use by `@mdx-js/monaco` and `@mdx-js/language-server`.
+It’s not intended for external usage.
+
+## Install
+
+This package is not published yet.
+
+## Use
+
+This package is intended for internal use only.
+If you do want to use this package for something else, please to let us know
+your use case in a [discussion][].
+
+## API
+
+This package exports the identifier `createMDXLanguageService`.
+There is no default export.
+
+### `createMDXLanguageService(ts, host[, plugins])`
+
+Create a [TypeScript][] language service that can handle [MDX][].
+
+#### Options
```

Comment:
```suggestion
#### Parameters
```


The word `options` to me is a thing that is optional, and has key/value things. These are parameters IMO